pulsar-common/src/main/java/org/apache/pulsar/common/mutable/AtomicMutableBoolean.java

```diff
@@ -0,0 +1,165 @@
+package org.apache.pulsar.common.mutable;
+
+import org.apache.commons.lang3.BooleanUtils;
+import org.apache.commons.lang3.mutable.MutableBoolean;
+
+public class AtomicMutableBoolean extends MutableBoolean {
```

Comment:
why do we need the class?

Reply:
can we just pass null to OpAddEntry when `setAddEntryTimeoutSeconds ` is disabled?

Reply:
> can we just pass null to OpAddEntry when setAddEntryTimeoutSeconds  is disabled?

Since the type of it is a wrap object, set to `null` is confusing. For example there is a variable typed `Optional<Object>,` and it has been set to `null` 😂